course_inventory/inventory.py

```diff
@@ -182,6 +182,31 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
     return course_df
 
 
+def gather_account_data_from_api(account_ids: Sequence[int]) -> pd.DataFrame:
+    logger.info('** gather_account_data_from_api')
+    url_ending_with_scope = f'{API_SCOPE_PREFIX}/accounts/'
+
+    account_dicts = []
+    for account_id in account_ids:
+        logger.info(f'Account number {account_id}')
```

Comment:
The message can say something like 'Fetching data for account {account_id}`

Reply:
I wasn't sure if this was necessary given that I'm logging the function name which describes this. Could I just add one logging message about this `for` loop that says "Fetching account data"?

Reply:
I think I've resolved your concerns here with [this commit](https://github.com/tl-its-umich-edu/course-inventory/pull/151/commits/25f0043f5d438f9c33aba909bd3b8a9abcad1788).